client/my-sites/plan-features/style.scss

```diff
@@ -10,15 +10,15 @@ $plan-features-sidebar-width: 272px;
 
 .is-section-plans .plan-features__mobile {
 	display: block;
-	@media ( min-width: $plan-features-sidebar-width + 480px ) {
```

Comment:
the spaces before and after the parentheses should remain, I'm not sure why our stylelint file doesn't include https://stylelint.io/user-guide/rules/function-parentheses-space-inside/